### PR TITLE
Update readme template and add example

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/registry/create/_index.md [hunk 1]
```diff
@@ -1149,12 +1152,80 @@ _For example:_
...
+## Build and Run
+
+To use this module, follow these instructions to [add a module from the Viam Registry](https://docs.viam.com/registry/configure/#add-a-modular-resource-from-the-viam-registry) and select the `viam:base:agilex-limo` model from the [`agilex-limo` module](https://app.viam.com/module/viam/agilex-limo).
+
+## Configure your AgileX LIMO base
 
+> [!NOTE]  
+> Before configuring your base, you must [create a robot](https://docs.viam.com/manage/fleet/robots/#add-a-new-robot).
+
+Navigate to the **Config** tab of your robot’s page in [the Viam app](https://app.viam.com/). Click on the **Components** subtab and click **Create component**. Select the `base` type, then select the `agilex-limo` model. Enter a name for your base and click **Create**.
```
Comment (npentrel) [inline docs/registry/create/_index.md:1174]:
```suggestion
Navigate to the **Config** tab of your robot’s page in [the Viam app](https://app.viam.com/). Click on the **Components** subtab and click **Create component**. Select the `base` type, then select the `agilex-limo` model. Click **Add module**, then enter a name for your base and click **Create**.
```

File: docs/registry/create/_index.md [hunk 2]
```diff
@@ -1149,12 +1152,80 @@ _For example:_
...
+
+{{% /tab %}}
+{{% tab name="Example" %}}
+
+````md
+# AgileX LIMO Driver
+
+This module implements the base driver for the [AgileX LIMO](https://global.agilex.ai/education/4) base to be used with [`viam-server`](https://docs.viam.com/). This driver supports differential, ackermann, and omni directional steering modes over the serial port.
+
+## Build and Run
```
Comment (sguequierre) [inline docs/registry/create/_index.md:1165]:
```suggestion
## Build and run
```
To keep headings rule like docs

File: docs/registry/create/_index.md [hunk 3]
```diff
@@ -1149,12 +1152,80 @@ _For example:_
...
+
+To use this module, follow these instructions to [add a module from the Viam Registry](https://docs.viam.com/registry/configure/#add-a-modular-resource-from-the-viam-registry) and select the `viam:base:agilex-limo` model from the [`agilex-limo` module](https://app.viam.com/module/viam/agilex-limo).
+
+## Configure your AgileX LIMO base
 
+> [!NOTE]  
+> Before configuring your base, you must [create a robot](https://docs.viam.com/manage/fleet/robots/#add-a-new-robot).
+
+Navigate to the **Config** tab of your robot’s page in [the Viam app](https://app.viam.com/). Click on the **Components** subtab and click **Create component**. Select the `base` type, then select the `agilex-limo` model. Click **Add module**, then enter a name for your base and click **Create**.
```
Comment (sguequierre) [inline docs/registry/create/_index.md:1174]:
```suggestion
Navigate to the **Config** tab of your machine’s page in [the Viam app](https://app.viam.com/). Click on the **Components** subtab and click **Create component**. Select the `base` type, then select the `agilex-limo` model. Click **Add module**, then enter a name for your base and click **Create**.
```
Did change in template

File: docs/registry/create/_index.md [hunk 4]
```diff
@@ -1149,12 +1152,80 @@ _For example:_
...
+This module implements the base driver for the [AgileX LIMO](https://global.agilex.ai/education/4) base to be used with [`viam-server`](https://docs.viam.com/). This driver supports differential, ackermann, and omni directional steering modes over the serial port.
+
+## Build and Run
+
+To use this module, follow these instructions to [add a module from the Viam Registry](https://docs.viam.com/registry/configure/#add-a-modular-resource-from-the-viam-registry) and select the `viam:base:agilex-limo` model from the [`agilex-limo` module](https://app.viam.com/module/viam/agilex-limo).
+
+## Configure your AgileX LIMO base
 
+> [!NOTE]  
+> Before configuring your base, you must [create a robot](https://docs.viam.com/manage/fleet/robots/#add-a-new-robot).
```
Comment (sguequierre) [inline docs/registry/create/_index.md:1172]:
```suggestion
> Before configuring your base, you must [create a machine](https://docs.viam.com/manage/fleet/robots/#add-a-new-robot).
```

File: docs/registry/create/_index.md [hunk 5]
```diff
@@ -1149,12 +1152,80 @@ _For example:_
...
+
+```json
+{
+  "drive_mode": "<ackermann|differential|omni>",
+  "serial_path": "<your-serial-path>"
+}
 ```
 
+> [!NOTE]  
+> For more information, see [Configure a Robot](https://docs.viam.com/manage/configuration/).
```
Comment (sguequierre) [inline docs/registry/create/_index.md:1186]:
```suggestion
> For more information, see [Configure a Machine](https://docs.viam.com/manage/configuration/).
```

File: docs/registry/create/_index.md [hunk 6]
```diff
@@ -1149,12 +1152,80 @@ _For example:_
...
+```
+
+```json
+{
+  "drive_mode": "omni",
+  "serial_path": "/dev/ttyTHS1"
+}
 ```
+
+## Next Steps
```
Comment (sguequierre) [inline docs/registry/create/_index.md:1212]:
```suggestion
## Next steps
```

File: docs/registry/create/_index.md [hunk 7]
```diff
@@ -1149,12 +1152,80 @@ _For example:_
...
+
+- To test your base, go to the [**Control** tab](https://docs.viam.com/manage/fleet/robots/#control).
+- To write code against your base, use one of the [available SDKs](https://docs.viam.com/program/).
+- To view examples using a base component, explore [these tutorials](https://docs.viam.com/tutorials/).
+
+## Local Development
```
Comment (sguequierre) [inline docs/registry/create/_index.md:1218]:
```suggestion
## Local development
```

**Review by sguequierre — CHANGES_REQUESTED** [review]

File: docs/registry/create/_index.md [hunk 8]
```diff
@@ -1079,13 +1079,16 @@ To provide usage instructions for any modular resources in your module, you shou
...
 If you follow [the instructions to generate module scaffolding for a Python module](#optional-generate-your-python-module-scaffolding), the template file is generated for you with the environment variables filled in.
 
+{{< tabs >}}
+{{% tab name="Template" %}}
+
 ````md
 # <INSERT NAME> modular resource
 
-This module implements the [<INSERT API NAMESPACE> <INSERT API NAME> API](https://github.com/<INSERT API NAMESPACE>/<INSERT API NAME>-api) in a <INSERT MODEL> model.
+This module implements the <INSERT API NAMESPACE> <INSERT API NAME> API in a <INSERT MODEL> model.
```
Comment (sguequierre) [inline docs/registry/create/_index.md:1091]: nice

File: docs/registry/create/_index.md [hunk 9]
```diff
@@ -1108,13 +1111,13 @@ To use this module, follow these instructions to [add a module from the Viam Reg
 Navigate to the **Config** tab of your machine's page in [the Viam app](https://app.viam.com/).
 Click on the **Components** subtab and click **Create component**.
 Select the `<INSERT API NAME>` type, then select the `<INSERT MODEL>` model.
-Enter a name for your <INSERT API NAME> and click **Create**.
+Click **Add module**, then enter a name for your <INSERT API NAME> and click **Create**.
```
Comment (sguequierre) [inline docs/registry/create/_index.md:1114]: Good catch

**Review by sguequierre — APPROVED** [review]

**Pull request merged**